src/components/Button/Button.js

```diff
@@ -41,9 +57,10 @@ Button.propTypes = {
 };
 
 Button.defaultProps = {
-  color: 'black',
-  width: '50px',
-  clickData: null
+  backgroundColor: null,
+  borderRadius: null,
```

Comment:
Same comment as above. Why null and not a meaningful value? what happens when this prop is not provided ?